src/cloud/amazon/instance.rs

```diff
@@ -1,32 +1,33 @@
 pub mod collector {
     use aws_config::meta::region::RegionProviderChain;
-    use aws_sdk_ec2::model::{Instance, Tag};
-    use aws_sdk_ec2::{Client, Error, Region};
+    use aws_sdk_ec2::types::Filter;
+    use aws_sdk_ec2::{Client, Error, config::Region};
     use crate::cloud::definition::AmazonCollection;
 
-    async fn match_instances(client: &Client) -> Result<(AmazonCollection, AmazonCollection), Error> {
-        let resp = client.describe_instances().send().await?;
-
+    async fn match_instances(client: &Client) -> Result<AmazonCollection, Error> {
+        // ["running", "pending", "shutting-down", "terminated", "stopped", "stopping"] are all the
+        // instance states, only grab active or soon to be active ones.
+        let filter = Filter::builder()
```

Comment:
use a filter to only grab running instances, might be desirable to add a config setting for this later.